### PR TITLE
New recipe for .dot files

### PR DESCRIPTION
**Pull request by vapniks**

### CODE REVIEW

File: recipes/dot/Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# ______________________________________________________________________________
+
+DIRDOT = $(shell dir *.dot)
+
+DIROUT.DOT = $(DIRDOT:.dot=.out.dot)
+
+all: clean Makefile $(DIROUT.DOT)
+
+%.out.dot: %.dot
+	echo "add command here"
```
Comment (abo-abo) [inline recipes/dot/Makefile]: Can you remove this file from the PR? It doesn't do anything.

File: recipes/dot/to-png/Makefile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+%.png: %.dot
+	dot -Tpng "$^" > "$@"
+	echo $@ >> provide
+
+clean:
+	rm -f *.png provide
+
+# Insert the install command here.
+# e.g. sudo apt-get install ffmpeg
+install-tools:
```
Comment (abo-abo) [inline recipes/dot/to-png/Makefile]: This should be:

    sudo apt-get install graphviz

Comment (vapniks) [inline recipes/dot/to-png/Makefile]: it is

File: recipes/dot/Makefile (hunk 1 above)
Comment (vapniks) [inline recipes/dot/Makefile]: done

**Comment by abo-abo** [comment]
Thanks.

**Pull request closed without merging**